nectar/src/maker.rs

```diff
@@ -160,64 +139,31 @@ impl Maker {
         }
     }
 
-    /// Decide whether we should proceed with order,
-    /// Confirm with the order book
-    /// Re & take & reserve
     pub fn process_taken_order(
```

Comment:
This should probably be renamed at some point :)

Reply:
Yes, I'll leave it like that for now. There is a todo on the matching function of `strategy::AllIn`, we can improve this interface when tackling the todo.